packages/in_app_purchase/lib/src/in_app_purchase/in_app_purchase_connection.dart

```diff
@@ -233,3 +233,33 @@ abstract class InAppPurchaseConnection {
     return _instance;
   }
 }
+
+/// Which platform the request is on.
```

Comment:
Now the produce response also uses this, so product_details also needs this. We can have product_details import purchase_details but it doesn't seem right either. After renaming the object in a follow up PR it should look better.
Another way to make this PR look better is to create another object for product_details, but i feel like that's an overkill since all it serves is just to make this particular PR look better.

Reply:
Ah got it. How it currently is LGTM.